### ISSUE TITLE
[BUG] UI not showing parameters

### ISSUE THREAD

**Issue by lucmos** [mlflow/mlflow#2411]
### System information
- **Have I written custom code**: yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Manjaro
- **MLflow installed from (source or binary)**: binary, poetry
- **MLflow version (run ``mlflow --version``)**: 1.6
- **Python version**: 3.7.5

### Describe the problem
Parameters are not displayed correctly in the new UI.

Usually, I use this format to log parameters, to semantically group them:

```
model.class = MyModel
model.loss_fun = mse
optimizer.class = Adam
optimizer.learning_rate = 0.0001
...
```

With the new UI, the parameters are not shown:

![image](https://user-images.githubusercontent.com/11019190/74137444-1da7ec80-4bf0-11ea-9232-c2e79b02a10c.png)

![image](https://user-images.githubusercontent.com/11019190/74137488-32848000-4bf0-11ea-82e1-79c0c330547b.png)

In each run (or in the compact view) the parameters are correctly shown:
![image](https://user-images.githubusercontent.com/11019190/74137510-403a0580-4bf0-11ea-9d23-c051ff6695c1.png)

### Code to reproduce issue
Log one parameter in the form `component.parameter`

### Other info
Would be great if the dot would create parameters group (exactly as it has been done for the parameters or tags categories)

**Comment by harupy** [comment]
@lucmos 
This bug is fixed by #2397.

**Comment by dbczumar** [comment]
Thanks @lucmos and @harupy. I'm going to close this since it has been fixed in the `master` branch!
